app/models/api_docs/service.rb

```diff
@@ -93,6 +95,13 @@ def specification
   end
 
   private
+
+  def service_belongs_to_account
```

Comment:
See https://github.com/3scale/porta/pull/62/files#r222188417

Reply:
Ok, better let's keep the conversation in one single place 😄 😉

Reply:
done 😉